res/css/photon/checkbox.css

```diff
@@ -6,8 +6,6 @@
   position: relative;
   padding: 0;
   border: 1px solid var(--grey-90-a30);
-  -webkit-appearance: none;
-  -moz-appearance: none;
```

Comment:
This is [property-no-vendor-prefix](https://stylelint.io/user-guide/rules/list/property-no-vendor-prefix/). I hesitated about that one, but then thought that autoprefixer would do a better job than us anyway.